src/OrchardCore.Modules/OrchardCore.OpenId/Startup.cs

```diff
@@ -204,6 +199,17 @@ public override void ConfigureServices(IServiceCollection services)
             });
         }
 
+        [RequireFeatures("OrchardCore.Deployment")]
```

Comment:
Needs to be outside the `ServerStartup` class.

And in this case it would be `[RequireFeatures("OrchardCore.Deployment", OpenIdConstants.Features.Server)]` to trigger the startup.

Same below with validation